lib/internal/Magento/Framework/App/FeedFormatsInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\App;
+
+interface FeedFormatsInterface
+{
+    const DEFAULT_FORMAT = 'rss';
```

Comment:
What I propose is to add two constants to Magento\Framework\App\FeedFactoryInterface

```php
namespace Magento\Framework\App;
 
 interface FeedFactoryInterface
 {
     const FEED_FORMAT_RSS = 'rss';
     const FEED_FORMAT_ATOM = 'atom';

     const FEED_OUTPUT_FORMAT_XML = 'xml';
    ....
 }
```